relayer/pkg/chainlink/erc20/client_test.go

```diff
@@ -57,7 +57,7 @@ func TestERC20Client(t *testing.T) {
 					out = []byte(`{"result":["0x1"]}`)
 				case starknetutils.GetSelectorFromNameFelt("balance_of").String():
 					// latest transmission details response
-					out = []byte(`{"result":["0x0"]}`)
```

Comment:
can you leave a comment here to indicate why there is two hex strings rather than just one?